admin/src/sagas/adminSaga.js

```diff
@@ -0,0 +1,77 @@
+import {
+  takeEvery,
+  select,
+  all,
+  call,
+  put,
+  takeLatest,
+} from "redux-saga/effects";
+import {
+  GET_ADMIN_REQUEST,
+  UPDATE_ADMIN_FAILURE,
+  UPDATE_ADMIN_REQUEST,
+  UPDATE_ADMIN_SUCCESS,
+  DELETE_ADMIN_REQUEST,
+  DELETE_ADMIN_SUCCESS,
+} from "../actions/constant";
+import {
+  getAdminFailure,
+  getAdminSuccess,
+  updateAdminFailure,
+  updateAdminSuccess,
+  deleteAdminFailure,
+  deleteAdminSuccess
+} from "../actions";
+import axiosInstance from "../helper/axios";
+
+const getAccount = (state) => state.admin;
+
+const fetchAccount = async (query) => {
+  const response = await axiosInstance.post(
```

Comment:
sao lại dùng post để fetch e

Reply:
bên api BE viết hơi chuối bắt dùng post anh ạ